Eatery Blue/UI/EateryCard/EaterySmallCardView.swift

```diff
@@ -2,62 +2,102 @@
+
+    // MARK: - Properties (View)
 
     private let imageView = UIImageView()
     private let favoriteButton = ContainerView(pillContent: UIImageView())
     private let titleLabel = UILabel()
+    private let stackView = UIStackView()
+
+    // MARK: - Properties (Data)
+
+    private var cancellables = Set<AnyCancellable>()
+    private var eatery: Eatery?
+    static let reuse = "EaterySmallCardViewReuseId"
+
+    // MARK: - Init
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
         setUpSelf()
-        setUpConstraints()
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
+
     func configure(eatery: Eatery) {
-        imageView.kf.setImage(with: eatery.imageUrl)
+        self.eatery = eatery
         titleLabel.text = eatery.name
-        setUpFavoriteButton(eatery: eatery)
+        configureImageView()
+        setUpFavoriteButton()
+        configureSubtitleLabels()
     }
 
     private func setUpSelf() {
+        contentView.backgroundColor = .white
+        contentView.clipsToBounds = true
+        contentView.layer.cornerRadius = 8
+        layer.shadowRadius = 4
+        layer.shadowOffset = CGSize(width: 0, height: 4)
+        layer.shadowColor = UIColor.Eatery.shadowLight.cgColor
+        layer.shadowOpacity = 0.25
+
         addSubview(imageView)
         setUpImageView()
 
         addSubview(titleLabel)
         setUpTitleLabel()
 
         addSubview(favoriteButton)
+
+        addSubview(stackView)
+        setUpStackView()
+
+        setUpFavNotification()
+        setUpConstraints()
     }
 
     private func setUpImageView() {
         imageView.contentMode = .scaleAspectFill
         imageView.clipsToBounds = true
         imageView.layer.cornerRadius = 8
+        imageView.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
     }
 
     private func setUpTitleLabel() {
-        titleLabel.font = .preferredFont(for: .subheadline, weight: .semibold)
+        titleLabel.font = .preferredFont(for: .headline, weight: .semibold)
         titleLabel.textColor = UIColor.Eatery.black
-        titleLabel.numberOfLines = 2
-        titleLabel.textAlignment = .center
+        titleLabel.numberOfLines = 1
+        titleLabel.textAlignment = .left
     }
 
-    private func setUpFavoriteButton(eatery: Eatery) {
-        favoriteButton.content.contentMode = .scaleAspectFill
-        favoriteButton.content.image = UIImage(named: "FavoriteSelected")
-        
+    private func setUpFavoriteButton() {
+        guard let eatery else { return }
+
+        let coreDataStack = AppDelegate.shared.coreDataStack
+        let metadata = coreDataStack.metadata(eateryId: eatery.id)
+        if metadata.isFavorite {
+            self.favoriteButton.content.image = UIImage(named: "FavoriteSelected")
+        } else {
+            self.favoriteButton.content.image = UIImage(named: "FavoriteUnselected")
+        }
```

Comment:
Since it is just one string difference, I would opt for a ternary expression rather than an if-else block:

`self.favoriteButton.content.image = UIImage(named: "Favorite\(metadata.isFavorite ? "Selected" : "Unselected)")`